src/phoenix/server/api/routers/v1/evaluations.py

```diff
@@ -204,3 +251,42 @@ def _read_sql_span_evaluations_into_dataframe(
         connectable,
         index_col="span_id",
     )
+
+
+def _read_sql_document_evaluations_into_dataframe(
+    connectable: Connectable,
+    project_name: str,
+) -> DataFrame:
+    """
+    This function inputs a synchronous connection to pandas.read_sql since
+    it does not support async connections.
+
+    For more information, see:
+
+    https://stackoverflow.com/questions/70848256/how-can-i-use-pandas-read-sql-on-an-async-connection
+    """
+    return pd.read_sql(
+        select(
+            models.DocumentAnnotation,
+            models.DocumentAnnotation.document_index.label("document_position"),
```

Comment:
@RogerHYang Makes me think we should rename this column `document_position` in the table.

Reply:
yea, it'll be less error-prone given the backward-compatibility requirements

Reply:
#2913